src/test/java/org/gridsuite/modification/server/BuildTest.java

```diff
@@ -449,14 +550,6 @@ public void runBuildTest() throws Exception {
         assertEquals("v1", network.getLine("newLine").getTerminal1().getVoltageLevel().getId());
         assertEquals("v2", network.getLine("newLine").getTerminal2().getVoltageLevel().getId());
         assertEquals(4., network.getLine("newLine").getB1(), 0.1);
-        assertEquals("v1", network.getTwoWindingsTransformer("new2wt").getTerminal1().getVoltageLevel().getId());
```

Comment:
Why do you remove these tests
Move them in the `TwoWindingsTransformerCreationTest` class